python/paddle/fluid/backward.py

```diff
@@ -1242,7 +1244,8 @@ def append_backward(loss,
             p_g_list6 = fluid.backward.append_backward(loss=avg_loss, parameter_list=all_weights, no_grad_set=set(all_weights))
 
     """
-    assert isinstance(loss, framework.Variable)
+    check_type(loss, 'loss', (framework.Variable),
```

Comment:
I think you can write `framework.Variable` or `(framework.Variable, )` instead of `(framework.Variable)`

Reply:
Thanks, done.